packages/turbo/android/src/main/java/com/reactnativeturbowebview/RNSession.kt

```diff
@@ -1,102 +1,156 @@
 package com.reactnativeturbowebview
```

Comment:
Should we consider renaming this to be more in-line with the package name(s)?

Reply:
As discussed earlier, we will probably rename the whole library so let's leave it for now.